src/generators/micropython/leaphy_original.ts

```diff
@@ -0,0 +1,77 @@
+import { MicroPython } from "../micropython";
+
+function getCodeGenerators(micropython: MicroPython) {
+    micropython.forBlock["leaphy_original_set_led"] = function (block) {
+        micropython.addInclude("rgbled", "from leaphymicropython.actuators.rgbled import RGBLed\n");
+        
+        micropython.addDeclaration("led_init", "led = RGBLed(6, 7, 9)\n", true, 90);
+        
+        const red = micropython.valueToCode(block, "LED_RED", 0) || "0";
+        const green = micropython.valueToCode(block, "LED_GREEN", 0) || "0";
+        const blue = micropython.valueToCode(block, "LED_BLUE", 0) || "0";
+        
+        return `led.set_color(${red}, ${green}, ${blue})\n`;
+    };
+
+
+
+    micropython.forBlock["digital_read"] = function (block) {
+        micropython.addInclude("machine", "from machine import Pin\n");
+        const pin = block.getFieldValue("PIN");
+        
+        micropython.addDeclaration(`pin_${pin}`, `pin_${pin} = Pin(${pin}, Pin.IN)`, false);
+        
+        return [`pin_${pin}.value()`, 0];
+    };
+
+    micropython.forBlock["analog_read"] = function (block) {
+        micropython.addInclude("read_analog", "from leaphymicropython.utils.pins import read_analog\n");
+        const pin = block.getFieldValue("PIN");
+        
+        // Arduino to GPIO pin mapping
+        const pinMapping: Record<string, number> = {
```

Comment:
This pin mapping should not go into this function. There are 2 other places where pins are defined (in micropython lib + robot definitions)